samples/microphone/transcribe_streaming_infinite.py

```diff
@@ -195,45 +193,42 @@ def listen_print_loop(responses, stream):
         transcript = result.alternatives[0].transcript
 
         result_seconds = 0
-        result_nanos = 0
+        result_micros = 0
 
         if result.result_end_time.seconds:
             result_seconds = result.result_end_time.seconds
 
-        if result.result_end_time.nanos:
-            result_nanos = result.result_end_time.nanos
+        if result.result_end_time.microseconds:
+            result_micros = result.result_end_time.microseconds
```

Comment:
This cannot be correct:  the [docs for `google.protobuf.Duration`](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) show only a `nanos` field.